README.md

```diff
@@ -1,26 +1,34 @@
 <!-- BEGIN_TF_DOCS -->
 # terraform-module-cloud-gcp-kubernetes-cluster
 
-This Terraform module deploys everything you need in GCP to get a kubernetes cluster up and running. This repo should be used in the context of deploying with an [admiral](https://github.com/glueops/admiral)
+This Terraform module deploys everything you need in GCP to get a kubernetes cluster up and running. This repo should be used in the context of deploying with the [admiral](https://github.com/glueops/admiral) repository.
 
 ## Prerequisites
 
 ### Project Setup
 
-1. Create a **new** GCP Project with the following APIs enabled:
+1. Create a **new** GCP Project with the following APIs enabled: [`cloudresourcemanager.googleapis.com`, `compute.googleapis.com`].  These APIs can be enabled in Cloud Shell using:
 
+```bash
+gcloud services enable cloudresourcemanager.googleapis.com compute.googleapis.com
 ```
-cloudresourcemanager.googleapis.com
-compute.googleapis.com
-```
 
-2. Service Quota increase: `In-use IP addresses (default is 8)`. Have this increased to 64. \_Note: this increase should be immediate, otherwise contact GCP support/account maangement\_
+2. Service Quota increase: `In-use IP addresses (default is 8)` has too low of a default value for deploying Kubernetes and quota should be increased to 64 to create headroom in available IPs.
+Edit the quota for `In-use IP addresses` in the [quota page](https://console.cloud.google.com/iam-admin/quotas) or with the following `gcloud` command.
+
+```bash
+gcloud alpha services quota update \
+    --service=compute.googleapis.com --consumer=projects/antoniostacos-gcp-nonprod \
+    --metric=compute.googleapis.com/global_in_use_addresses \
+    --unit=1/{project} --value=64
+```
+\_Note: If the increase isn't immediate or you receive an error, contact GCP support/account management\_
 
 ![Screenshot of Quota Increase](https://user-images.githubusercontent.com/6570292/210277244-f3a75d06-763f-4bdc-805e-4f8bd3c77ad5.png)
 
 ### Service Account
 
-1. Create a service account and download the json key file, save the file as `creds.json`
+1. [Create a service account](https://console.cloud.google.com/iam-admin/serviceaccounts/create) and [download the json key file](https://console.cloud.google.com/iam-admin/serviceaccounts/details/101612329871957262389/keys), save the file as `creds.json`
```

Comment:
If the plan is to have them auth with gcloud then we could probably have them run the commands to create this with the gcloud cli too

Reply:
created issue: https://github.com/GlueOps/terraform-module-cloud-gcp-kubernetes-cluster/issues/14